apigee_edge.services.yml

```diff
@@ -123,3 +130,6 @@ services:
   apigee_edge.converter.user_developer:
     class: Drupal\apigee_edge\UserDeveloperConverter
     arguments: ['@config.factory', '@entity_type.manager', '@plugin.manager.apigee_field_storage_format', '@apigee_edge.converter.field_attribute']
+
+  apigee_edge.token_storage:
```

Comment:
I was a little bit sceptical whether this needs to be a service but at least this way we can swap it and replace it with a more secure storage solution if needed on customer projects.
Should not this be called `apigee_edge.oauth_token_storage` instead? Or `apigee_edge.authentication.oauth_token_storage`? "token_storage" is way too general I think.